sklearn/feature_selection/rfe.py

```diff
@@ -146,6 +149,13 @@ def fit(self, X, y):
 
         support_ = np.ones(n_features, dtype=np.bool)
         ranking_ = np.ones(n_features, dtype=np.int)
+
+        if step_score:
+            # Determine the number of subsets of features
+            self.scores_ = np.zeros(
```

Comment:
I initially use a list, but I have to get the size the ```scores``` in RFECV beforehand anyway , because ```scores``` in RFECV cannot be a list to do ```scores += rfe.scores_```.  So it does not make much difference, I think.